sky/skylet/providers/azure/node_provider.py

```diff
@@ -322,6 +322,73 @@ def _create_node(self, node_config, tags, count):
         create_or_update = get_azure_sdk_function(
             client=self.resource_client.deployments, function_name="create_or_update"
         )
+        poller = create_or_update(
+            resource_group_name=resource_group,
+            deployment_name=vm_name,
+            parameters=parameters,
+        )
+        poller.wait()
+
+        # pylint: disable=import-outside-toplevel
+        from sky.clouds.service_catalog import azure_catalog
+
+        instance_type = node_config["azure_arm_parameters"].get("vmSize", "")
+        accs = azure_catalog.get_accelerators_from_instance_type(instance_type)
+        if accs is None or "A10" not in accs:
+            return
+
+        # Configure driver extension for A10 GPUs
+        create_result = poller.result().as_dict()
```

Comment:
We should merge the parameters for A10 to original parameters to avoid two `create_or_update` which can cause significant overhead? Please help profile it.

Reply:
Good point! I benchmarked and it reduces the provision time from `15m15s` to `14m36s`.